code/modules/mob/living/say.dm

```diff
@@ -73,7 +73,7 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 
 		if(AmountSluring())
 			if(robot)
-				S.message = slur(S.message, list("@", "!", "#", "$", "%", "&", "?"))
+				S.message = slur(S.message, list("@", "!", "#", "$", "%", "&", "?"), robot)
```

Comment:
```suggestion
				S.message = slur(S.message, list("@", "!", "#", "$", "%", "&", "?"), TRUE)
```
Whoops, github seems to have forgotten my suggestion on my last review, but this `robot` here should be `TRUE`